### PR TITLE
feat: added logging to inbox events and aggregated measure data handler

### PR DESCRIPTION
**Pull request by MadsDue**
## Description

Added logging in isolated scope for the HostedService AggregatedTimeSeriesServiceBusWorker
## References

* #0000

### CODE REVIEW

**Review by johevemi — APPROVED** [review]

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs [hunk 1]
```diff
@@ -54,9 +54,10 @@ await foreach (var p in GetInternalAsync(sql, batch.PeriodStart.ToInstant(), bat
 
     public async IAsyncEnumerable<CalculationResult> GetAsync(CalculationResultQuery query)
     {
-        var sql = CreateRequestSql(query);
-        await foreach (var p in GetInternalAsync(sql, query.StartOfPeriod, query.EndOfPeriod))
+        var sqlStatement = CreateRequestSql(query);
+        await foreach (var p in GetInternalAsync(sqlStatement, query.StartOfPeriod, query.EndOfPeriod))
             yield return p;
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs:59]: What is `p`? How about this?
```suggestion
        await foreach (var result in GetInternalAsync(sqlStatement, query.StartOfPeriod, query.EndOfPeriod))
            yield return result;
```

File: source/dotnet/wholesale-api/Common/Logging/LoggerExtensions.cs [hunk 2]
```diff
@@ -30,4 +31,12 @@ public static class LoggerExtensions
     {
         logger.LogDebug("Entering {MethodName} in {FilePath}:{LineNo}", methodName, sourceFile, lineNumber);
     }
+
+    /// <summary>
+    /// Utility method to log entire objects.
+    /// </summary>
+    public static string Dump(this object o)
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/Common/Logging/LoggerExtensions.cs:38]: Dumping to where? How about `ToJsonString` or similar?

File: source/dotnet/wholesale-api/Events/Events.Application/Workers/AggregatedTimeSeriesServiceBusWorker.cs [hunk 3]
```diff
@@ -39,13 +41,19 @@ public class AggregatedTimeSeriesServiceBusWorker : BackgroundService, IAsyncDis
         _serviceBusProcessor = serviceBusClient.CreateProcessor(options.Value.WHOLESALE_INBOX_MESSAGE_QUEUE_NAME);
         _aggregatedTimeSeriesRequestHandler = aggregatedTimeSeriesRequestHandler;
         _logger = logger;
+
+        _serviceName = GetType().Name;
+        _loggingScope = new Dictionary<string, object> { ["HostedService"] = _serviceName };
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/Events/Events.Application/Workers/AggregatedTimeSeriesServiceBusWorker.cs:46]: Please use the LoggingScope from [here](https://github.com/Energinet-DataHub/geh-core/pull/237).

File: source/dotnet/wholesale-api/Events/Events.Application/Workers/AggregatedTimeSeriesServiceBusWorker.cs [hunk 4]
```diff
@@ -39,13 +41,19 @@ public class AggregatedTimeSeriesServiceBusWorker : BackgroundService, IAsyncDis
...
     public override async Task StopAsync(CancellationToken cancellationToken)
     {
-        _logger.LogDebug("Stopping the service bus subscription");
-        await _serviceBusProcessor.CloseAsync(cancellationToken).ConfigureAwait(false);
-        await base.StopAsync(cancellationToken).ConfigureAwait(false);
+        using (_logger.BeginScope(_loggingScope))
+        {
+            await _serviceBusProcessor.CloseAsync(cancellationToken).ConfigureAwait(false);
+            await base.StopAsync(cancellationToken).ConfigureAwait(false);
+            _logger.LogWarning("{Worker} has stopped at {Time}", _serviceName, DateTimeOffset.Now);
```
[the remark this answers is not in the document]
Comment (MadsDue) [reply source/dotnet/wholesale-api/Events/Events.Application/Workers/AggregatedTimeSeriesServiceBusWorker.cs:55]: I was inspired of how you did it in the RepeatingTrigger.cs

File: source/dotnet/wholesale-api/Events/Events.Application/Workers/AggregatedTimeSeriesServiceBusWorker.cs (hunk 3 above)
Comment (MadsDue) [reply source/dotnet/wholesale-api/Events/Events.Application/Workers/AggregatedTimeSeriesServiceBusWorker.cs:46]: I can find any docs on how to use this package, only a describtion on how to register it in a web app. It looks like the solution currently only use it for adding scopes to asp.net requests :S Besides it looks like I should inject a RootLoggingScope. 

I find inspiration in the RepeatingTrigger for impl logging scope.   

Maybe we should have a chat about this.

File: source/dotnet/wholesale-api/Events/Events.Application/Workers/AggregatedTimeSeriesServiceBusWorker.cs (hunk 4 above)
[the remark this answers is not in the document]
Comment (BjarkeMeier) [reply source/dotnet/wholesale-api/Events/Events.Application/Workers/AggregatedTimeSeriesServiceBusWorker.cs:55]: Be cautious with that 😆

File: source/dotnet/wholesale-api/Events/Events.Application/Workers/AggregatedTimeSeriesServiceBusWorker.cs (hunk 3 above)
Comment (BjarkeMeier) [reply source/dotnet/wholesale-api/Events/Events.Application/Workers/AggregatedTimeSeriesServiceBusWorker.cs:46]: Thinking again - it probably makes sense that Mandalorian introduces the package in the domain. So don't worry about this.

**Pull request merged**